lab_share_lib/config/rabbit_config.py

```diff
@@ -0,0 +1,13 @@
+from dataclasses import dataclass
+from typing import Dict, Type
+
+from lab_share_lib.config.rabbit_server_details import RabbitServerDetails
+from lab_share_lib.processing.base_processor import BaseProcessor
+
+
+@dataclass
+class RabbitConfig:
+    consumer_details: RabbitServerDetails
+    consumed_queue: str
```

Comment:
Do we need to add any validations for the presence of this `consumed_queue` variable? I assume, given that it's not made `Optional`, it's mandatory.

Reply:
I don't personally think we should be validating app config.  Data that comes in from outside your app at runtime should be validated (or known good) but these are values set by the developer before app launch.